packages/logger/src/index.ts

```diff
@@ -58,18 +62,27 @@ const getNamespace = (namespace: string): string => {
 
 // tslint:disable-next-line: variable-name
 export const NSlogger = (namespace: string = ''): ILogger => {
-  const namespaceModified = getNamespace(namespace);
-
   Object.assign(instance.defaultMeta, {
     ...instance.defaultMeta,
-    namespace: namespaceModified,
+    namespace: getNamespace(namespace),
   });
 
+  const params = (msg, severity) =>
+    ENV.nodeEnv !== 'development' ? [...msg, { severity, component: 'arbitrary-property' }] : msg;
+
   return {
-    info: instance.info.bind(instance),
-    warn: instance.warn.bind(instance),
-    debug: ENV.logLevel === 'debug' ? getDebugger(namespaceModified) : () => {},
-    error: instance.error.bind(instance),
+    info: (...msg) => {
+      instance.info.bind(instance)(...params(msg, 'Info'));
+    },
+    warn: (...msg) => {
+      instance.warn.bind(instance)(...params(msg, 'Warn'));
+    },
+    debug: (...msg) => {
+      instance.debug.bind(instance)(params(msg, 'Debug'));
```

Comment:
I switched out the `debug` package implementation for the `winston` package implementation as the `winston` package has a google cloud logging add-on package made by Google themselves and `debug` does not.

Reply:
@KnlnKS what does that package enable us to do?

`debug` package:

>allows you to toggle the debug output for different parts of your module as well as the module as a whole.

Meaning, we can control whether to show debug logs or not based on some namespace. Does winston/GCP logging add-on have this feature?